### ISSUE TITLE
image: bpfman container image - bpfman fails to start

### ISSUE THREAD

**Issue by Billy99** [bpfman/bpfman#1318]
Loading the current `quay.io/bpfman/bpfman:v0.5.3` image in a KIND cluster, the bpfman container fails to start:

```
$ kubectl logs -n bpfman bpfman-daemon-4pnk8 -c bpfman
./bpfman-rpc: /lib64/libc.so.6: version `GLIBC_2.39' not found (required by ./bpfman-rpc)
```

**Comment by Billy99** [comment]
bpfman images are built in github actions. The binary is built using `.github/wokflows/image-build.yml` which is currently using `ubuntu:latest`. Then the image is build by copying the binary into a container. The base image for the container is `redhat/ubi9-minimal`, which used glibc 2.34. Recently, ubuntu:latest change from ubuntu 22.04 (glibc 2.34) to
ubuntu 24.04 (glibc 2.39). So now bpfman won't run in a redhat/ubi9-minimal base image.

**Comment by Billy99** [comment]
Fixed by: #1314
